### DISCUSSION TITLE
Convert PSD with clothing mockups (warps) into scripts

### DISCUSSION THREAD

**Discussion by Sublime1981** [ImageMagick/ImageMagick#5990] [Consulting]
For a rather larger project than usual, we're looking to get at least 100 PSD files, that consist of clothing images with warps applied to them, converted into scripts using ImageMagick. Looking to hire someone for this specific project, which most likely will be ongoing. 

If you're interested you can send me a DM (not sure if that's the right way, as it's my first time here on Github).

**Comment by fmw42** [comment]
Please clarify further. Are you saying you need to create the PSD files or they will be the input. Will the warps need to be applied to images or are you saying the images you will get will have these warps.  Can you post an example of the input and output.
